dub.json

```diff
@@ -56,17 +59,29 @@
                 "source/agora/node/*",
                 "source/scpp/*",
                 "source/scpd/*",
-                "source/agora/consensus/protocol/*"
+                "source/agora/consensus/protocol/*",
+                "source/agora/cli/multi/*"
             ]
         },
         {
             "name": "unittest",
             "targetName": "agora-unittests",
-            "excludedSourceFiles": [ "source/agora/cli/client/main.d" ],
+            "excludedSourceFiles": [
+                "source/agora/cli/client/main.d"
+            ],
```

Comment:
```suggestion
            "excludedSourceFiles": [ "source/agora/cli/*/main.d" ],
```

Reply:
Ah I see you have `unittest` in main... Then wait before applying this one.
But in any case, the current diff is superfluous (nothing change but indentation).